inlong-common/src/main/java/org/apache/inlong/common/msg/InLongMsg.java

```diff
@@ -246,13 +246,12 @@ public boolean addMsg(String attr, ByteBuffer data) {
             return false;
         }
 
-        DataBuffer outputBuffer = attr2MsgBuffer.get(attr);
-        if (outputBuffer == null) {
-            outputBuffer = new DataBuffer();
-            attr2MsgBuffer.put(attr, outputBuffer);
+        DataBuffer outputBuffer = attr2MsgBuffer.computeIfAbsent(attr, key -> {
+            DataBuffer newBuffer = new DataBuffer();
             // attrlen + utflen + meglen + compress
-            this.datalen += attr.length() + 2 + 4 + 1;
-        }
+            this.datalen += key.length() + 2 + 4 + 1;
```

Comment:
This line is not the logic of the `mappingFunction`

Reply:
> This line is not the logic of the `mappingFunction`
Problem fixed